pkg/virt-controller/watch/pool_test.go

```diff
@@ -103,6 +106,7 @@ var _ = Describe("Pool", func() {
 			go vmiInformer.Run(stop)
 			go vmInformer.Run(stop)
 			go poolInformer.Run(stop)
+			go crInformer.Run(stop)
 			Expect(cache.WaitForCacheSync(stop, vmiInformer.HasSynced, vmInformer.HasSynced, poolInformer.HasSynced)).To(BeTrue())
```

Comment:
Shouldn't you also wait for the `crInformer` here?

Reply:
yes! fixed